packages/mapper/src/mapper.ts

```diff
@@ -560,19 +699,30 @@ export default class Mapper {
    */
   async insertMany(collection, docs, options: any = {}): Promise<any> {
     const dbOptions: any = {};
-    this.messageBus.emit('method:insertMany', collection._name, docs);
+    const db = collection._database._name;
+    const coll = collection._name;
 
     if ('writeConcern' in options) {
       dbOptions.writeConcern = options.writeConcern;
     }
 
+    this.messageBus.emit(
+      'mongosh:api-call',
+      {
+        method: 'insertMany',
+        class: 'Collection',
+        db, coll, arguments: { docs, options }
```

Comment:
I am keen to say let's not log the very many docs people will be inserting. Keeping it here for a sec, in case someone thinks otherwise.